.gitattributes

```diff
@@ -0,0 +1,17 @@
+# Omit during export.
+.github/ export-ignore
+docs/ export-ignore
+tests/ export-ignore
+controls/scss/ export-ignore
+.* export-ignore
+*.sh export-ignore
+*.md export-ignore
+*.txt export-ignore
+*.xml export-ignore
+*.dist export-ignore
+*.json export-ignore
+*.lock export-ignore
+LICENSE export-ignore
+Gruntfile.js export-ignore
+example.php export-ignore
+LICENSE export-ignore
```

Comment:
Hmmm actually we'll need to change quite a few things here.
The dev branch is a lot different than previous versions since it's on v4.0 and the whole structure has changes, everything is being migrated to separate composer packages etc.

`*.json` will have to removed... The googlefonts are on json files so we don't want to remove those :smile: 
`*.txt` will have to be removed.
`LICENCE` is required so it will also have to be removed
`controls/scss` no longer exists

The above are things that will have to be removed from the file. But there's some things that will have to be added too... 
Each control/package has its own scss file, so it may be a good idea to ignore `*.scss`
packages also have `package.json` and `composer.json` files that will have to be ignored.
I assume `Gruntfile.js` will ignore all grunt files regardless of their location? If not we may need to add `vendor/kirki-framework/**/Gruntfile.js` in there too